### DISCUSSION TITLE
Shield.IO

### DISCUSSION THREAD

**Discussion by ElectronicsArchiver** [pulsar-edit/.github#100] [Pulsar Features]
In short, https://shields.io/ badges for PPM as APM had.

**Comment by confused-Techie** [comment]
@ElectronicsArchiver this is already and open feature request for the backend on our [project board](https://github.com/orgs/pulsar-edit/projects/2).

**Reply by ElectronicsArchiver** [reply]
Oh, didn't see that, cool.
